systest/scripts/run_neutron_lbaas.sh

```diff
@@ -39,6 +37,16 @@ bash -c "tox -e tempest -c tox.ini --sitepackages -- \
 
 cd ${NEUTRON_LBAAS_DIR}
 
+# Navigate to the root of the repo, where the tox.ini file is found
```

Comment:
this block of code needs to be removed, line 30 to 38 is running the driver tempest target.